src/hubbleds/components/generic_state_components/stage_one/guideline_choose_row.vue

```diff
@@ -1,15 +1,16 @@
 <template>
-  <v-alert
+  <scaffold-alert
     color="info"
     class="mb-4 mx-auto"
     max-width="800"
     elevation="6"
+    header-text="Choose a Row"
+    @back="() => { state.marker = 'sel_gal3'; }"
+    @next="() => { state.marker = 'mee_spe1'; }"
+    next-content="Select a galaxy from your table."
+    :can-advance="(state) => state.spec_viewer_reached"
+    :state="state"
   >
```

Comment:
replace next-content with named slot

Reply:
Good catch, thanks! The pre-next content was a prop in the initial version of this and I obviously forgot to change this one.